pkg/bytecode/code_test.go

```diff
@@ -0,0 +1,63 @@
+package bytecode
+
+import (
+	"testing"
+
+	"evylang.dev/evy/pkg/assert"
+)
+
+func TestMake(t *testing.T) {
+	tests := []struct {
+		op       Opcode
+		operands []int
+		expected []byte
+	}{
+		{OpConstant, []int{65534}, []byte{byte(OpConstant), 255, 254}},
+	}
+	for _, tt := range tests {
+		instruction, err := Make(tt.op, tt.operands...)
+		assert.NoError(t, err)
+		assert.Equal(t, tt.expected, instruction)
+	}
+}
+
+func TestInstructionsString(t *testing.T) {
+	instructions := []Instructions{
+		mustMake(t, OpConstant, 2),
+		mustMake(t, OpConstant, 65535),
+	}
+	expected := "0000 OpConstant 2\n0003 OpConstant 65535\n"
```

Comment:
this may read easier as:

```
expected := `
0000 OpConstant 2
0003 OpConstant 6553
`[1:]
```

just mentioning because there might be more like this coming.
up to you - don't feel strongly about this.

Reply:
this also applies to various examples below. Maybe the single line solution for 2 things is preferable to the 4 line one, but I think for 3 or more things splitting over several lines with raw strings may be better.

Reply:
In our dev branch we ended up with a mix of styles (https://github.com/evylang/evy/blob/compiler/pkg/vm/vm_test.go#L13 vs https://github.com/evylang/evy/blob/compiler/pkg/compiler/compiler_test.go#L18)

This PR is all consistently single line, which I think is fine for now and we can swap to multiline when/if needed - wdyt?